charts/kubewarden-controller/values.yaml

```diff
@@ -19,6 +19,9 @@ affinity: {}
 
 # Policy Server settings
 policyServer:
+  default:
+    enabled: true
```

Comment:
I would prefer to not have `policyServer.default.enabled`. IIUC, if a user sets it to false, it will be the empty string in the controller.

Reply:
Maybe I'm misunderstanding, but I see it for providing the string that will be used as default PolicyServer, when instantiating policies that don't define it.
The controller was deployed with `--default-policy-server foo`. This `foo` gets saved in the controller, and its the value that the controller uses for policies that don't specifcy a `Spec.PolicyServer`.

But this is unrelated with instantiating a default PolicyServer or not (regardless of its name, `default`, or `foo`). As this should happen in the new chart `kubewarden-defaults`.

Reply:
yes, this will change with the new chart. However in this PR default `PolicyServer` will not be created if it's not enabled because of this if condition https://github.com/kubewarden/helm-charts/pull/70/files#diff-f665e14eea6253e710a96b26e781644c7a6d2d4d2d20308549c931f72c9ab08aR1

Reply:
It's for both. If you set `policyServer.default.enabled=false` you won't get a default policy server created at all.

Reply:
So, if I understand correctly @viccuad you propose:

```yaml
policyServer:
  defaultName: <something>
```

And in the controller, `manager --default-policy-server-name <something>`. Right?

WRT the structure of the values, I think is better structured to have:

```yaml
policyServer:
  default:
    enabled: true|false
    name: <something>
```

So it's possible to install without a default policy server in an easy way, as @raulcabello commented.

WRT the controller arguments I don't have a strong opinion, but I don't think `--default-policy-server` vs `--default-policy-server-name` makes a huge difference. I don't have a reason to reject `--default-policy-server-name`, but being longer my question would be, why `--default-policy-server` is not clear enough.

Reply:
if we want to merge this before the release of `kubewarden-defaults` chart, I would drop `policyServer.default.enabled` to keep current functionality. And when we release `kubewarden-defaults`, the logic will be there.

Reply:
Let's wait for `kubewarden-defaults` chart, and I will rebase this on top of that, making that part more clear.

Reply:
The `kubewarden-defaults` is merged. ;-)